layouts/partials/json-schema/resolve-additional-types.html

```diff
@@ -36,7 +38,11 @@
     {{/*
       Add the object we were passed into the $additional_objects array
```

Comment:
hrm. It feels like `$additional_objects` is a confusing name for this thing... it's *all* the objects including the root. Maybe we should rename it.